java/jdbc/src/main/java/com/flipkart/vitess/util/Constants.java

```diff
@@ -98,6 +98,24 @@ private Constants() {
         public static final String KEYSPACE = "KEYSPACE";
         public static final String USERNAME = "userName";
         public static final String EXECUTE_TYPE = "executeType";
+
+        public static final String USE_SSL = "useSSL";
+
+        public static final String KEYSTORE = "keyStore";
+        public static final String KEYSTORE_PASSWORD = "keyStorePassword";
+        public static final String KEY_ALIAS = "keyAlias";
+        public static final String KEY_PASSWORD = "keyPassword";
+        public static final String TRUSTSTORE = "trustStore";
+        public static final String TRUSTSTORE_PASSWORD = "trustStorePassword";
+        public static final String TRUSTSTORE_ALIAS = "trustAlias";
+
+        public static final String KEYSTORE_FULL = "javax.net.ssl.keyStore";
+        public static final String KEYSTORE_PASSWORD_FULL = "javax.net.ssl.keyStorePassword";
```

Comment:
Bug... I was incorrectly using KEY_PASSWORD_FULL twice in `VitessVTGateManager`.  Thanks!